ver/us/symbol_addrs.txt

```diff
@@ -2362,8 +2362,8 @@ IES_ITEM_137 = 0x8008DDB4; // rom:0x691B4
 gItemEntityScripts = 0x8008DDD4; // rom:0x691D4
 gItemIconRasterOffsets = 0x8008E3D8; // rom:0x697D8
 gItemIconPaletteOffsets = 0x8008E94C; // rom:0x69D4C
-D_8008EEC0 = 0x8008EEC0; // rom:0x6A2C0
-D_8008EEF0 = 0x8008EEF0; // rom:0x6A2F0
+PartnerMenuIndex_to_ID = 0x8008EEC0; // rom:0x6A2C0
```

Comment:
I'm not a big fan of the underscores in the names. Could we do `PartnerMenuIndexToID` and `PartnerIDToMenuIndex` ?

Reply:
PartnerMenuIndexToId or PartnerMenuIndexIdMap I'd say